packages/react-components/review/beta/react-components.api.md

```diff
@@ -1395,8 +1395,13 @@ export interface VideoGalleryProps {
 export interface VideoGalleryRemoteParticipant extends VideoGalleryParticipant {
     isSpeaking?: boolean;
     screenShareStream?: VideoGalleryStream;
+    // @beta
+    state?: VideoGalleryRemoteParticipantState;
 }
 
+// @beta
+export type VideoGalleryRemoteParticipantState = 'Connecting' | 'Ringing' | 'Hold';
```

Comment:
does calling SDK use `hold` or `onHold` when it referes to something being on hold?